Makefile

```diff
@@ -22,6 +22,11 @@ build-and-push-images:
 		cd scripts && ./build-and-push-images.sh
 .PHONY: build-and-push-images
 
+## Build and push images to local registry.
+build-and-push-java-image:
+		cd scripts && ./build-and-push-java-image.sh
+.PHONY: build-and-push-java-image
+
```

Comment:
This Makefile has changed a lot since the beginning of this PR - please grab the latest changes from `master`. You don't need now to create a `build-and-push-java-image` target anymore :)

Reply:
ok, I'll remove this function